### PR TITLE
Adds authorization for Content and Content Files controllers.

### LINKED ISSUE #152: Add authorization to Content and Content Files controllers
Currently, they have `skip_verify_authorized`

### PR DESCRIPTION
**Pull request by justinlittman**
closes #152

### CODE REVIEW

File: app/controllers/content_files_controller.rb [hunk 1]
```diff
@@ -2,15 +2,19 @@
 
 # Controller for a Work content file
 class ContentFilesController < ApplicationController
-  # Skipping authorization for now
-  skip_verify_authorized only: %i[show edit update destroy]
```
Comment (mjgiarlo) [inline app/controllers/content_files_controller.rb:6]: Can you replace the changes to the action methods below with:

```
  verify_authorized only: %i[show edit update destroy]
```

❓

File: app/controllers/contents_controller.rb [hunk 2]
```diff
@@ -2,17 +2,21 @@
 
 # Controller for a Work contents (files)
 class ContentsController < ApplicationController
-  # Skipping authorization for now
-  skip_verify_authorized only: %i[show edit update]
```
Comment (mjgiarlo) [inline app/controllers/contents_controller.rb:6]: Same question as other controller.

**Review by mjgiarlo — APPROVED** [review]
Approve pending consideration of AP question in controllers

File: app/controllers/content_files_controller.rb (hunk 1 above)
Comment (justinlittman) [reply app/controllers/content_files_controller.rb:6]: We already do that here: https://github.com/sul-dlss/hungry-hungry-hippo/blob/main/app/controllers/application_controller.rb#L11

`verify_authorized` makes sure that `authorize!` is called; it doesn't do the calling itself.

Comment (mjgiarlo) [reply app/controllers/content_files_controller.rb:6]: Ah, thank you!

**Pull request merged**